File: auto-docs/executables/nodejs/bar_marker_array.js
var plotly = require('plotly')('TestBot', 'r1neazxo9w')

var data = [
  {
    x: [1, 2, 3, 4], 
    y: [5, 4, -3, 2], 
    marker: {color: ["#447adb", "#447adb", "#db5a44", "#447adb"]}, 
    type: "bar"
  }
];

var graph_options = {filename: "bar-marker-array", fileopt: "overwrite", auto_open: "false"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});

File: auto-docs/executables/r/bar_marker_array.r
library(plotly)
py <- plotly(username='TestBot', key='r1neazxo9w')

data <- list(
  list(
    x = c(1, 2, 3, 4), 
    y = c(5, 4, -3, 2), 
    marker = list(color = c("#447adb", "#447adb", "#db5a44", "#447adb")), 
    type = "bar"
  )
)
response <- py$plotly(data, kwargs=list(filename="bar-marker-array", fileopt="overwrite"))
url <- response$url
